src/Livewire/Servers.php

```diff
@@ -14,6 +16,10 @@
 #[Lazy]
 class Servers extends Card
 {
+    use InteractsWithTime;
+
+    public int|string|null $ignoreAfter = null;
+
```

Comment:
Livewire does not support `DateInterval` as a prop value, so I've excluded it from the type here.

Due to the way Livewire serializes values, we also cannot use `now()->addSeconds(60)` in the prop value.